osi_common.proto

```diff
@@ -569,6 +707,23 @@ message BaseMoving
     // The polygon is defined counter-clockwise.
     //
     repeated Vector2d base_polygon = 7;
+
+    // Sub-divisions of the overall bounding box of the \c BaseMoving object.
+    //
+    // The bounding box sections can include side mirrors, cargo, etc. for 
+    // vehicles, as well as body-part sections for pedestrians.
+    //
+    // \note When one or more \c BoundingBox s are associated to a 
+    // \c BaseMoving , the expectation is that all sections are contained
+    // within the bounds of the \c #dimension .
```

Comment:
Saying that bounding box sections can include side mirrors here but then saying they might not be included within the bounding box dimensions in the respective enum description (see line 459) seems kind of ambiguous and confusing to me.
Shouldn't we be clear about if we want them to be included or not? Otherwise some people include side mirrors, others don't...

I think the main questions are:
- Would it break backwards compatibility if we include side mirrors in the bounding box?
- If yes, do we want to change this for 4.0?

Based on that I would rephrase the descriptions to explain why it is as it is and maybe also state the goal for the next major release (if there is any).